ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/ClientImpl.java

```diff
@@ -358,7 +360,9 @@ public BatchedQueryResult executeQuery(
 
     makePostRequest(
         KSQL_ENDPOINT,
-        new JsonObject().put("ksql", "drop connector " + name + ";"),
+        new JsonObject()
+            .put("ksql", "drop connector " + name + ";")
+            .put("sessionVariables", sessionVariables),
```

Comment:
We also need this for create connector, right? I don't see the change in this commit.

Reply:
That was in the previous commit.